package.json

```
@@ -32,7 +32,8 @@
     "@radix-ui/react-tabs": "1.0.3",
     "@radix-ui/react-tooltip": "1.0.5",
     "dayjs": "1.11.7",
-    "lodash.merge": "4.6.2"
+    "lodash.merge": "4.6.2",
+    "react-map-gl": "^7.0.21"
```

Would be nice to keep using fixed versions, but no blocker

```suggestion
    "react-map-gl": "7.0.21"
```